libcontainer/container_linux.go

```diff
@@ -368,13 +369,13 @@ func (c *linuxContainer) start(process *Process) error {
 			if err != nil {
 				return err
 			}
-			for i, hook := range c.config.Hooks.Poststart {
-				if err := hook.Run(s); err != nil {
-					if err := ignoreTerminateErrors(parent.terminate()); err != nil {
-						logrus.Warn(err)
-					}
-					return newSystemErrorWithCausef(err, "running poststart hook %d", i)
+			s.Status = configs.Running
```

Comment:
Why are we explicitly setting the state to `Running` here? There is a call to `c.currentOCIState()` a few lines above and we are overwriting that state here.

Reply:
Good catch! As part of the copy paste for these calls, I also copied the fact that we were manually setting the state.

Should we be setting the state anywhere?